dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -43,6 +43,12 @@ const linkStyles = (palette: Palette) => css`
 			border-bottom: 1px solid ${palette.hover.keyEventLink};
 		}
 	}
+
+	&:hover > div {
+		text-decoration: underline;
+		text-underline-offset: 3px;
+		text-decoration-color: ${palette.text.keyEvent};
```

Comment:
we can consolidate this line into line 48 by using the `text-decoration: underline ${palette.text.keyEvent}`; shorthand which would allow us to remove this extra property.

Reply:
Yeah it sure is, a little extra reading here if you're interested https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration